src/constructors/test/__snapshots__/injectGlobal.test.js.snap

```diff
@@ -0,0 +1,14 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`injectGlobal should extract @import rules into separate style tags 1`] = `
+"<style data-styled-components=\\"\\">
+/* sc-component-id: sc-global-3616276198 */
+html{padding:1px;}
+/* sc-component-id: sc-a */
+.sc-a {} .b{color:green;}
+/* sc-component-id: sc-global-1433896746 */
+html{color:blue;}</style>
+<style data-styled-components=\\"\\">
+/* sc-component-id: sc-global-1433896746-import */
+@import url('bla');</style>"
```

Comment:
Shouldn't this style tag be above the other style tags? Or is it only about the first thing in a certain style tag?

Reply:
@mxstbr I could enforce it to come before the style tag that contains the `@import` actually. I'm not sure whether we'd want that, since it's mostly meant for fonts only anyway.

But I could modify the `makeStyleTag` method to allow me to inject elements before another

Reply:
Not sure, I mean if it works that's totally fine!